docroot/themes/humsci/humsci_basic/templates/components/global-footer.html.twig

```diff
@@ -0,0 +1,51 @@
+{#
+  /**
+  * @file
+  * Global Footer Component.
+  *
+  * Stanford Universal Footer as defined at https://identity.stanford.edu/web-mobile.html#build-something-new
+  * This component should be added into the footer region of your site below site-specific footer section, if any.
+  * The entire footer can be made to "stick" to the bottom of the browser window by including the @sticky-footer mixin.
+  */
+#}
+
+{% if template_path_logo is empty -%}
+  {%- set template_path_logo = "/themes/humsci/humsci_basic/templates/components/logo.html.twig" -%}
+{%- endif %}
+
+<section class="hb-global-footer su-global-footer {{ modifier_class }}">
```

Comment:
can we make this section a `footer` instead to be semantic? I don't think it affects styles

Reply:
let's also add the `hb-page-width` class here :)

Reply:
I chose to update the `<div id="footer">` in `page.html.twig` to `<footer id="footer">` instead. Let me know what you think. I'm not sure what `{{ page.footer }}` is. Maybe it's the local footer?